packages/wpcom-checkout/src/checkout-line-items.tsx

```diff
@@ -909,11 +927,12 @@ function UpgradeCreditInformation( { product }: { product: ResponseCartProduct }
 	if ( isBiennially( product ) || isTriennially( product ) ) {
 		return (
 			<>
-				{ translate( 'Upgrade Credit: %(discount)s applied in first term only', {
+				{ translate( '%(humanReadableCode)s : %(discount)s applied in first term only', {
```

Comment:
This one is not really your change, but in `applied in first term only` wouldn't we just use `year` here as well? A year is an exact unit of time, where term could be rather open ended 🤔

Reply:
Also note, if we change it from term to year, we have a number of comments that also use `term`.

Reply:
This is running inside the 2- and 3-year conditional, so the discount applies to a term greater than a year. We could break this down into "applied in first {two|three} years only".

Reply:
I have now split it into two separate if conditions for  "applied in first {two|three} years only".